File: Demizon.Maui/ApiConfig.cs
namespace Demizon.Maui;

/// <summary>
/// Centrální konfigurace API URL.
/// Pro fyzický telefon přes Wi-Fi změň IP na aktuální adresu PC (ipconfig).
/// </summary>
internal static class ApiConfig
{
#if ANDROID
    public const string BaseUrl = "http://192.168.1.100:5272";
#else
    public const string BaseUrl = "http://localhost:5272";
#endif
}

File: demizon_flutter/lib/core/api_config.dart
/// Centrální konfigurace API URL.
///
/// Přebíráno z `Demizon.Maui/ApiConfig.cs`. Hodnotu lze přebít při buildu:
/// `flutter run --dart-define=DEMIZON_API_BASE_URL=http://192.168.0.10:8083`
class ApiConfig {
  const ApiConfig._();

  static const String baseUrl = String.fromEnvironment(
    'DEMIZON_API_BASE_URL',
    defaultValue: 'https://demizon-production.up.railway.app',
  );
}
